### PR TITLE
added schedule function

### PR DESCRIPTION
**Pull request by weining-li**
starting time increases by increments of 15 minutes

no tests yet :<

#15

### CODE REVIEW

**Review by allygator — DISMISSED** [review]

**Comment by ToanQVu** [comment]
Should this be in an issue branch instead of the develop branch?

**Comment by allygator** [comment]
Oh, whoops. yes it should.

File: src/lambda/schedule.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  let mealsBuffer = 120
+  let totalSchedule = []
+  let visited = []
+  for (let i = 0; i < freeTime.length; i++)
+  {
+      visited.push(false)
+  }
+
+  do
+  {
```
Comment (ToanQVu) [inline src/lambda/schedule.js:72]: What is up with those indentations?

File: src/lambda/schedule.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  calcSchedule(freeTime, mealsBuffer, timeBetweenMeals, visited, 0, freeTime.length, [], totalSchedule)
+  timeBetweenMeals -= 15
+  mealsBuffer += 30
+  } while (totalSchedule.length == 0 && mealsBuffer <= 210)
+
+  return totalSchedule
+}
+
+calcSchedule = (freeTime, buffer, timeBetween, visited, currentIndex, size, feedTime, totalSchedule) => {
```
Comment (ToanQVu) [inline src/lambda/schedule.js:81]: Do we need `const` before the definition of the function?

**Review by ToanQVu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**